### ISSUE TITLE
Missing label/control associations for accessibility

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ericpromislow/tabhunter#1]
```
What steps will reproduce the problem?
1.
2.
3.

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?


Please provide any additional information below.

 In the Tabhunter panel, the textboxes and listbox are missing associations
with their proper labels. As a result, screen readers for the blind cannot
pick up the correct label for each textbox or other control like listboxes etc.

In the XUL accessibility authoring guidelines at
http://developer.mozilla.org/en/docs/Accessible_XUL_Authoring_Guidelines,
this is specified (specify the control attribute on the xul:label element
to pint to the ID attribute's value of the control it is labelling). Please
Use xul:label, not xul:description for this.
```

Original issue reported on code.google.com by `marco.z...@gmail.com` on 9 Jul 2008 at 3:33
